sos/report/plugins/powerpc.py

```diff
@@ -22,15 +22,15 @@ class PowerPC(Plugin, IndependentPlugin):
 
     def setup(self):
         try:
-            with open(self.path_join('/proc/cpuinfo'), 'r') as fp:
-                contents = fp.read()
-                ispSeries = "pSeries" in contents
-                isPowerNV = "PowerNV" in contents
+            with open('/proc/cpuinfo', 'r', encoding='UTF-8') as file:
```

Comment:
The removed `self.path_join`: can't be the plugin called from a non-default `sysroot`, what would cause different file opened?

Reply:
Good catch!

Reply:
Thanks! Fixed in both places (ssh.py)